test/functional/vimscript/msgpack_functions_spec.lua

```diff
@@ -20,17 +20,18 @@ describe('msgpack*() functions', function()
 
   -- Regression test: msgpack_list_write was failing to write buffer with zero
   -- length.
-  obj_test('are able to dump and restore {"file": ""}', {{file=''}})
+  obj_test('are able to dump and restore {"file": ""}', { { file = '' } })
   -- Regression test: msgpack_list_write was failing to write buffer with NL at
   -- the end.
-  obj_test('are able to dump and restore {0, "echo mpack"}', {{0, 'echo mpack'}})
-  obj_test('are able to dump and restore "Test\\n"', {'Test\n'})
+  obj_test('are able to dump and restore {0, "echo mpack"}', { { 0, 'echo mpack' } })
+  obj_test('are able to dump and restore "Test\\n"', { 'Test\n' })
   -- Regression test: msgpack_list_write was failing to write buffer with NL
   -- inside.
-  obj_test('are able to dump and restore "Test\\nTest 2"', {'Test\nTest 2'})
+  obj_test('are able to dump and restore "Test\\nTest 2"', { 'Test\nTest 2' })
   -- Test that big objects (requirement: dump to something that is bigger then
   -- IOSIZE) are also fine. This particular object is obtained by concatenating
   -- 5 identical shada files.
+  -- stylua: ignore
   local big_obj = {
     1, 1436711454, 78, {
```

Comment:
This is data so added `stylua: ignore`. Nice thing about `stylua: ignore` is that it is local to the scope.